packages/language/src/v1beta1/index.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var languageServiceApi = require('./language_service_api');
+var gax = require('google-gax');
+
+function v1beta1(options) {
+  options = options || {};
+  if (!('scopes' in options)) {
```

Comment:
I agree with customization as well, but in the case of scopes, won't the API refuse to work if the user changes them? I don't think we want to support that.